extensions/ql-vscode/src/remote-queries/view/FileCodeSnippet.tsx

```diff
@@ -24,8 +24,8 @@ const getSeverityColor = (severity: ResultSeverity) => {
 const replaceSpaceAndTabChar = (text: string) => text.replaceAll(' ', '\u00a0').replaceAll('\t', '\u00a0\u00a0\u00a0\u00a0');
 
 const Container = styled.div`
-  font-family: ui-monospace, SFMono-Regular, SF Mono, Menlo, Consolas, Liberation Mono, monospace;
-  font-size: x-small;
+  font-family: -apple-system, BlinkMacSystemFont, Segoe UI, Helvetica, Arial, sans-serif, Apple Color Emoji, Segoe UI Emoji;
```

Comment:
It looks like the font is no longer monospace. Is that intentional?

Reply:
The original issue wasn't very clear, but I think the user (rightly) pointed out that our monospace font in the results view was much harder to read than the monospace font in the VS Code editor. So we probably want to match that editor font.

I expect we can do this by using the VS Code variable: `font-family: var(--vscode-editor-font-family);` (warning, I haven't properly tested this 🙈 )

Reply:
> Yes, the original issue was:
> "The font used in the code snippet is quite small and narrowly spaced compared to the source files."
> The base styles don't use monospace:

Optional extra, if you want to fiddle around with the individual containers: While the code snippet should stay monospace, I could see the file link + message text looking nice in the base styles, e.g. something like this. What do you think? 🎨 

![image](https://user-images.githubusercontent.com/42641846/171493629-de205e57-a585-4b81-bdb9-d614bcaef07d.png)

(Feel free to ignore though. Updating the overall font and font size is already an improvement! 💅🏽 ) 

---------- 
Side note: you probably already know this, but it's helpful to poke around in the VS Code webview using `Developer: Open Webview Developer Tools` (I didn't know about this command for ages 😆 )

Reply:
Thanks, @shati-patel, I've updated the font family to use `var(--vscode-editor-font-family)` as you suggested and adjusted the size of `MessageText`. Both changes have taken effect. 

## Before

![Screenshot 2022-06-13 at 12 58 13](https://user-images.githubusercontent.com/1354439/173361962-8b622aef-5c50-4049-99fc-cd782afc5ea3.png)

## After

![Screenshot 2022-06-13 at 13 16 26](https://user-images.githubusercontent.com/1354439/173361981-01b2b50e-4ee2-4345-b90f-c27069da2e0c.png)

I didn't know about `Developer: Open Webview Developer Tools`. Thanks for teaching me that! It makes changing props a lot easier.

In terms of the title and file link, I believe that because they're inside the `Container` component, they're already inheriting the font size:

![Screenshot 2022-06-13 at 13 21 06](https://user-images.githubusercontent.com/1354439/173362145-19e6d62d-e834-47f9-9ba1-4bad67bf1db1.png)